### DISCUSSION TITLE
Any idea how to boot from aarch64.raw.xz instead of ISO?

### DISCUSSION THREAD

**Discussion by EchoEllet** [utmapp/UTM#6361] [UTM Help (macOS)]
I would like to thank the authors for their amazing work.

When downloading any Linux distro installer for the ARM® aarch64 systems instead of Intel and AMD x86_64 systems, usually it doesn't come as an iso file, for example for [Linux Fedora KDE](https://fedoraproject.org/spins/kde/download) or [Manjaro KDE](https://manjaro.org/download/)

![image](https://github.com/utmapp/UTM/assets/73608287/06ae4eba-55c9-49f2-b872-cf823c62a417)

![image](https://github.com/utmapp/UTM/assets/73608287/7c271860-f300-4615-8e7d-b25a42e88a08)

The reason I want the ARM aarch64 version instead of Intel and AMD x86_64 systems is so I can use the option `Virtualize` instead of `Emulate` in UTM

![image](https://github.com/utmapp/UTM/assets/73608287/ea152e82-e43b-4b5a-8619-be11566fb8fd)

When downloading it, it comes as `Fedora-KDE-40-1.14.aarch64.raw.xz`, extracting it to `Fedora-KDE-40-1.14.aarch64.raw`

![image](https://github.com/utmapp/UTM/assets/73608287/cc954f64-37c1-4444-99ba-7ec7f86afd09)

There is an option for `Boot ISO Image` and it accepts any kind of file, even `.txt` files, selecting either the downloaded file or the extracted file will not boot into the system

**Comment by EchoEllet** [comment]
Changing the image type from `CD/DVD (ISO) Image` to `Disk Image` in the settings

![image](https://github.com/utmapp/UTM/assets/73608287/17eb293e-c74a-4940-b2ad-5550d216bc3a)

Fixed the issue
![image](https://github.com/utmapp/UTM/assets/73608287/b51c6b56-1da5-43d5-bce1-66c3584a1f57)

Now I have a new issue
![image](https://github.com/utmapp/UTM/assets/73608287/c4c9b218-9ca4-4b26-ba8b-6d7253d98be7)

**Comment by appleseedexm** [comment]
I got this working with a `.raw` partition. Instead of using the usb disk, just import a new drive that is **non-removable** and select your **raw** file (resulting drive see screenshot). That drive was then easily bootable (change boot order by drive order in UTM or use `F2` during boot).

My usecase was resizing the boot partition of the UTM arch image (which is too small btw) and that worked flawlessly.

<img width="640" alt="Screenshot 2024-06-11 at 17 15 20" src="https://github.com/utmapp/UTM/assets/11321388/4e886372-f45d-4412-bcf5-66b720b95efe">

**Reply by alaz-aura** [reply]
Can confirm this answer works. Thanks @appleseedexm 👍 
<img width="1195" alt="Screenshot 2024-09-02 at 10 12 26 AM" src="https://github.com/user-attachments/assets/eea0184a-bf7f-429d-a2b7-b11a6db13f86">

**Reply by subhashhhhhh** [reply]
> I got this working with a `.raw` partition. Instead of using the usb disk, just import a new drive that is **non-removable** and select your **raw** file (resulting drive see screenshot). That drive was then easily bootable (change boot order by drive order in UTM or use `F2` during boot).
> 
> My usecase was resizing the boot partition of the UTM arch image (which is too small btw) and that worked flawlessly.
> 
> <img alt="Screenshot 2024-06-11 at 17 15 20" width="640" src="https://private-user-images.githubusercontent.com/11321388/338645395-4e886372-f45d-4412-bcf5-66b720b95efe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUyOTc2NzEsIm5iZiI6MTcyNTI5NzM3MSwicGF0aCI6Ii8xMTMyMTM4OC8zMzg2NDUzOTUtNGU4ODYzNzItZjQ1ZC00NDEyLWJjZjUtNjZiNzIwYjk1ZWZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTAyVDE3MTYxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI4MjZmZmJmOTc3MGEwY2Y4ZTdlNzg3NjQyNTYxMDE0MmQwODRmZmI5NjE1NmM3ZjNlMGFlZGU5Njk0YjNkNDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.51-U8BzL-OChuwbjIbG7GWjBWWB5e99LLDkUVCECsqw">

can i remove the drive after the first boot?

**Reply by appleseedexm** [reply]
> can i remove the drive after the first boot?

Yes, if you dont need that partition anymore and only want to boot your existing drive.
